apps/keyboard/test/unit/imes/latin/worker_test.js

```diff
@@ -527,14 +515,23 @@ suite('Latin worker', function() {
         successCount++;
 
         if (successCount === 4) {
+          worker.onmessage = null;
           next();
         }
       }
     };
   }
 
+  test('Worker should throw if no dictData is passed', function(done) {
+    worker.postMessage({ cmd: 'setLanguage', args: ['en_us', null] });
+    onWorkerMessage('error', function(data) {
+      assert.equal(data.cmd, 'error');
+      done();
```

Comment:
it would also be easier if we are Promise based:
```javascript
onWorkerMessage('error').then(
  () => assert.equal(data.cmd, 'error')
).then(done, done);
```